flipperdevices/flipperzero-firmware
# [FL-1299] iButton fix previous scene transition

# What's new

- Add new scene for file select

# Verification 

- Enter saved key menu. Push back. Verify next scene is list of saved keys
- Delete key. Verify next scene is list of saved keys

# Checklist (do not modify)
